### DISCUSSION TITLE
intake: can't access several variables from cmip6 archive

### DISCUSSION THREAD

**Discussion by MaxThomas90** [intake/intake-esm#503] [Q&A] [answered]
Hello,
I've noticed several variables are available to download from ESGF that cannot be accessed via the pangeo tools. I'm not sure if I'm doing something wrong, if this is by design, or if there's an issue to flag. I've put a short example below:

I need to access variable_id=sidmassth, source_id=HadGEM3-GC31-LL, experiment_id=ssp585, table_id=SImon
Looking on the ESGF archive I can see that the variable is there for my model and experiment.
![image](https://user-images.githubusercontent.com/56394971/180672223-79096cf8-eab6-405f-a3d1-3611f9377bdb.png)

when I try to import that variable intake can't find it. 
![image](https://user-images.githubusercontent.com/56394971/180672246-81241331-ce63-4808-b272-49e9020372da.png)

In fact, sidmassth doesn't appear in the list of HadGEM's SImon variables at all:
![image](https://user-images.githubusercontent.com/56394971/180672262-bf9c5bae-e686-46b5-a25f-182a6dcd028a.png)

Finally I make sure that intake is generally working by loading a variable that does appear under SImon
![image](https://user-images.githubusercontent.com/56394971/180672274-4d3e40e5-b272-4456-b56c-060cdee2f473.png)

**Comment by andersy005** [comment, marked as answer]
@MaxThomas90, this is not an intake-esm issue per se. The Pangeo-CMIP6 archive is a subset of the CMIP6 data hosted on the ESGF nodes. To request for additional data to be added to the Pangeo-CMIP6 archive, i recommend opening an issue here: https://github.com/pangeo-data/pangeo-cmip6-cloud/issues

**Comment by MaxThomas90** [comment]
Thanks. I've opened an issue there.
Max
